wp-content/themes/core/components/routes/unsupported_browser/unsupported_browser.php

```diff
@@ -50,10 +49,10 @@ class="browser-list__item-anchor"
 					   rel="external noopener"
 					   target="_blank">
 						<span class="browser-list__item-image">
-						  <img src="<?php echo $controller->legacy_browser_icon_chrome ?>"
-							alt="<?php echo esc_attr( __( 'Chrome browser logo', 'tribe' ) ) ?>" />
+						  <img src="<?php echo $c->legacy_browser_icon_chrome; ?>"
```

Comment:
Do we need `esc_url()` here?

Reply:
I don't think so, it does get escaped in the controller.